test/box-tap/gh-5388-lua-key-def-leak.test.lua

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env tarantool
```

Comment:
Why this file is named "gh-5388-lua-key-def-leak.test.lua" and not "gh-5388-lua-key_def-leak.test.lua"?
"test/test-run.py key_def" would not run it.

Reply:
I prefer to use the same word separator in such cases, because it is hard to visually spot the underscore in the series of dash separated words.

Reply:
I agree with Sergey here. I use that pattern of providing a part of test name very often to test things related to the module I am working on. Also 'def' is not a separate word. 'key_def' is a word, it can't be separated, because it is a name of the struct/module.

Reply:
'Key' and 'definition' are different words even despite that their composition forms a term. However you show the use case, when exact struct/module name is useful, so I changed it in the test name and the changelog entry name.

Reply:
(Marked as resolved.)